### PR TITLE
Added ability to edit license.

### PR DESCRIPTION
**Pull request by MrK191**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -774,10 +774,6 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <version>${jacoco.version}</version>
-                <configuration>
-                    <destfile>${basedir}/target/coverage-reports/jacoco-unit.exec</destfile>
-                    <datafile>${basedir}/target/coverage-reports/jacoco-unit.exec</datafile>
-                </configuration>
```
Comment (madryk) [inline pom.xml:780]: Why we commented this?

File: src/main/java/edu/harvard/iq/dataverse/license/LicenseListingPage.java [hunk 2]
```diff
@@ -51,6 +51,7 @@
 
     private LicenseDto freshLicense;
     private LicenseDto licenseForPreview;
+    private LicenseDto licenseForEditing;
```
Comment (madryk) [inline src/main/java/edu/harvard/iq/dataverse/license/LicenseListingPage.java]: 'licenseForEdit'?

**Review by madryk — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (MrK191) [reply pom.xml:780]: Intellij was highlighting it in red that "Element destfile/datafile is not allowed here" and coverage-reports doesn't exists in target folder, so those lines are not doing anything.

Comment (madryk) [reply pom.xml:780]: Ok. It seems that these properties are `destFile` and `dataFile` but if they worked before with default values then good that we get rid of it

**Review by madryk — APPROVED** [review]

**Review by madryk — APPROVED** [review]

**Pull request merged**